api/base/settings.py

```diff
@@ -41,13 +41,14 @@
 
     # 3rd party
     'rest_framework',
+    'rest_framework_swagger',
 )
 
 REST_FRAMEWORK = {
     'PAGE_SIZE': 10,
     'DEFAULT_RENDERER_CLASSES': (
-        'api.base.renderers.JSONAPIRenderer',
         'rest_framework.renderers.JSONRenderer',
+        'api.base.renderers.JSONAPIRenderer',
```

Comment:
Here's the issue I posted: https://github.com/marcgibbons/django-rest-swagger/issues/271